### PR TITLE
[Xthor] Feature xthor replace vostfr english

### PR DESCRIPTION
**Pull request by Tawagot0**
For the issue #8759

Modele:
firefox_ghK0caDjv5

Without the option checked:

firefox_KDXMmmC9mg

With the option:

firefox_tgih3UF9wH

### CODE REVIEW

File: src/Jackett.Common/Indexers/Xthor.cs [hunk 1]
```diff
@@ -10,6 +10,7 @@
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using Jackett.Common.Models;
+using Jackett.Common.Models.GitHub;
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/Xthor.cs:13]: you don't need this

File: src/Jackett.Common/Indexers/Xthor.cs [hunk 2]
```diff
@@ -230,6 +231,13 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                             torrent.name = regex.Replace(torrent.name, "$1" + ReplaceMulti + "$2");
                         }
 
+                        // issue #8759 replace vostfr and subfrench with English
+                        if (ConfigData.Vostfr.Value)
+                        {
+                            torrent.name = torrent.name.Replace("VOSTFR", "ENGLISH");
+                            torrent.name = torrent.name.Replace("SUBFRENCH", "ENGLISH");
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/Xthor.cs:238]: you can make both replacements in one line and remove the braces

Comment (ngosang) [reply src/Jackett.Common/Indexers/Xthor.cs]: I was thinking on:
```
if (ConfigData.Vostfr.Value)
    torrent.name = torrent.name.Replace("VOSTFR", "ENGLISH")
                                                  .Replace("SUBFRENCH", "ENGLISH");
```
but it was my fault for not being explicit. next time

**Comment by ngosang** [comment]
@Tawagot0 thank you! We have another issue open for this tracker. Could you take a look? #6392

**Comment by Tawagot0** [comment]
> 
> 
> @Tawagot0 thank you! We have another issue open for this tracker. Could you take a look? #6392

Yes sorry I had not read my emails!

**Pull request merged**